src/py42/_internal/clients/archive.py

```diff
@@ -10,6 +11,76 @@ def get_data_key_token(self, device_guid):
         data = {u"computerGuid": device_guid}
         return self._session.post(uri, data=json.dumps(data))
 
+    def get_single_archive(self, archive_guid):
+        """Gets single archive information by GUID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            archive_guid (int): The GUID for the archive.
+
+        Returns:
+            :class:`py42.response.Py42Response`: A response containing archive information.
+        """
+        uri = u"/api/Archive/{}".format(archive_guid)
+        return self._session.get(uri)
+
+    def get_page(self, page_num, page_size=None, **kwargs):
```

Comment:
Consider enumerating more arguments here. Perhaps not the deprecated ones.

Reply:
I considered updating it to `get_page(self, page_num, device_guid=None, user_uid=None, destination_guid=None, page_size=None, **kwargs):`, but the new arguments wouldn't get used because they're passed as `**kwargs`. 

Since `get_page` should only ever get one of the values `device_guid`, `user_uid`, or `destination_guid`, I'm not sure of the benefit (other than perhaps documentation), unless I've misunderstood something?

Reply:
Is this simply for documentation purposes? The reason I ask is because currently, the query parameters in `get_all_archives_from_value` are generated from `id_value` and `id_type`, like this;

```
params = {"{}".format(id_type): "{}".format(id_value)}
return get_all_pages(self.get_page, "archives", **params,)
```

If I add the expected parameters as arguments to `get_page` (similar to `UserClient`), `get_all_pages` will complain about receiving duplicate keyword parameters (since it's already being passed as a keyword parameter). Also `get_page` for `ArchiveClient` will only expect one query parameter since the API only supports a single ID.

What I thought was that I would just enumerate the expected arguments in `get_page`, add docstrings for them, but then omit them from `params` in `get_page`, but that doesn't seem like the right way. 

Do you have any suggestions? Or maybe I'm misunderstanding?

Reply:
Ah, I just meant other **kwargs, not the IDs

Reply:
Looks like there is a `srtKey` and a `srtDir` at least

Reply:
Perhaps `export`, which outputs a CSV. That may be useful down the line.

Let me be clear that this was just a suggestion and is not required